src/main/webapp/js/userMap.js

```diff
@@ -0,0 +1,104 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+    // based on example from https://github.com/markmarkoh/datamaps/blob/master/src/examples/highmaps_world.html
+    // Country code: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
+
+    var countriesObj = {};
+    var countriesArr = [];
```

Comment:
Using both makes the runtime O(n) and using only array makes runtime O(n^2) because you need to look for the correct element.